src/spikeinterface/core/generate.py

```diff
@@ -32,10 +32,9 @@ def generate_recording(
     set_probe: Optional[bool] = True,
     ndim: Optional[int] = 2,
     seed: Optional[int] = None,
-    mode: Literal["lazy", "legacy"] = "lazy",
 ) -> BaseRecording:
     """
-    Generate a recording object.
+    Generate a lazy recording object.
     Useful for testing for testing API and algos.
```

Comment:
```suggestion
    Useful for testing API and algos.
```
When one testing is not enough...